src/main/scala/viper/gobra/ast/internal/PrettyPrinter.scala

```diff
@@ -442,6 +443,7 @@ class DefaultPrettyPrinter extends PrettyPrinter with kiama.output.PrettyPrinter
 
   def showLit(l: Lit): Doc = l match {
     case IntLit(v, _) => v.toString
+    case StringLit(s) => s"${"\""}$s${"\""}"
```

Comment:
this seems quite complicated. Wouldn't `"\"" <> s <> "\""` work as well?

Reply:
Yes, that is a good suggestion 👍